python/py-brotli/Portfile

```diff
@@ -26,11 +26,9 @@ checksums           rmd160  cb6909cbcc7c0bb19b8e985dbd7fcbca1b651772 \
                     sha256  10973f4b4199eafa1d5735ef661ddb2ec2f97319ee9fd1824d4aabe08cff5265 \
                     size    512036
 
-python.versions     37 38 39 310 311 312
+python.versions     38 39 310 311 312
 
 if {$subport ne $name} {
-    depends_build-append port:py${python.version}-setuptools
```

Comment:
Are we sure setuptools is not required for Python >= 3.12?

Reply:
For Python 3.7 and above “python.pep517 yes” is set and it uses as default “python.pep517_backend setuptools”. 

So yes, it is still needed but automatically added by the “python” PortGroup, that’s why an explicit build dependency in the Portfile is redundant.

Reply:
Thanks for clarification, that totally makes sense. I was not aware the python port group adds it.